src/sage/databases/cubic_hecke_db.py

```diff
@@ -31,10 +31,24 @@
 of linear forms on the cubic Hecke algebra on at most four strands
 satisfying the Markov trace condition for its cubic Hecke subalgebras.
 
+To use the database, you need to install the optional package by the Sage
+command ::
+
+    sage -i database_cubic_hecke
```

Comment:
Ideally, whenever a `sage -i` line is shown, there should also be a link to the SPKG page such as https://doc.sagemath.org/html/en/reference/spkg/database_cubic_hecke.html#spkg-database-cubic-hecke;
for some of the packages there are install instructions for equivalent system packages

Reply:
Yes. Done.